config/checkstyle_checks.xml

```diff
@@ -551,6 +551,11 @@
     <module name="JavadocMethod">
       <property name="validateThrows" value="true"/>
     </module>
+    <module name="JavadocMethod">
+      <property name="id" value="privateMethodsJavadoc"/>
```

Comment:
Why 2 instances of the same check? Really we just lost the setting of private when we changed the default value for the scope.

Also, default scope is already private. It was the Missing method which was the issue.
https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocMethodCheck.java#L375

Reply:
removed